src/main/webapp/app/entities/programming-exercise/programming-exercise-options.model.ts

```diff
@@ -0,0 +1,6 @@
+import { ProgrammingExercise } from 'app/entities/programming-exercise/programming-exercise.model';
+
+export interface ProgrammingExerciseOptions {
```

Comment:
Do we really need this? 

`buildAndTestStudentSubmissionsAfterDueDate` is part of `ProgrammingExercise` and is serialized as JSON only here. I guess if we use another object / interface here, we might get problems with the serialization.

Or is there any reason why you need this?

Reply:
You are right, this was not meant to stay. I'll delete it.